src/tsserver/nodeServer.ts

```diff
@@ -1,3 +1,8 @@
+import childProcess from "child_process";
+import fs from "fs";
+import net from "net";
+import os from "os";
+import readline from "readline";

```

Comment:
Is there any risk of these becoming unconditional imports?

Reply:
Nope, this is the tsserver project which only runs in node.